.github/workflows/goreleaser.yml

```diff
@@ -0,0 +1,33 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
for sanity, maybe we want to ensure it only releases for tags that start with `v*`.